example/l3fwd/odp_l3fwd_run.sh

```diff
@@ -6,23 +6,27 @@
 # SPDX-License-Identifier:     BSD-3-Clause
 #
 
-PCAP_IN=`find . ${TEST_DIR} $(dirname $0) -name udp64.pcap -print -quit`
-PCAP_OUT="pcapout.pcap"
-PCAP_IN_SIZE=`stat -c %s ${PCAP_IN}`
-echo "using PCAP_IN = ${PCAP_IN}, PCAP_OUT = ${PCAP_OUT}"
+if  [ -f ./interface_env ]; then
+	. ./interface_env
+else
+        echo "BUG: unable to find interface_env!"
+        echo "interface_env has to be in current directory"
+        exit 1
+fi
+
+setup_interfaces
 
-./odp_l3fwd${EXEEXT} -i pcap:in=${PCAP_IN},pcap:out=${PCAP_OUT} \
-	    -r "10.0.0.0/24,pcap:out=${PCAP_OUT}" -d 30
+./odp_l3fwd${EXEEXT} -i $IF0,$IF1 -r "10.0.0.0/24,pcap:out=${PCAP_OUT}" -d 30
```

Comment:
This is still using linux-generic specific interface name `pcap:out=${PCAP_OUT}`.

Reply:
Missed to replace. Will take in next version.